src/tinyformat/README.md

```diff
@@ -0,0 +1,452 @@
+# tinyformat.h
+
+## A minimal type safe printf() replacement
```

Comment:
No need to add a third party readme. This will show up in all history/logs and greps.